packages/bundle-source/src/index.js

```diff
@@ -9,6 +9,10 @@ const DEFAULT_MODULE_FORMAT = 'getExport';
 const DEFAULT_FILE_PREFIX = '/bundled-source';
 const SUPPORTED_FORMATS = ['getExport', 'nestedEvaluate'];
 
+// eslint-disable-next-line no-useless-concat
+const IMPORT_RE = new RegExp('\\b(import)' + '(\\s*(?:\\(|/[/*]))', 'g');
```

Comment:
Does this say typescript import statements can start either with `import (name` or `import /name`?

I couldn't find a description that included anything other than the paren. Am I misunderstanding the vertical bar in the RE?

Reply:
I just took what is used in SES.  I don't understand about the slash either.